tools/run_tests/run_interop_tests.py

```diff
@@ -1029,5 +1096,8 @@ def aggregate_http2_results(stdout):
   dockerjob.finish_jobs([j for j in server_jobs.itervalues()])
 
   for image in docker_images.itervalues():
-    print('Removing docker image %s' % image)
-    dockerjob.remove_image(image)
+    if not args.manual_run:
+      print('Removing docker image %s' % image)
+      dockerjob.remove_image(image)
+    else:
+      print('Preserving docker image: %s' % image)
```

Comment:
I feel we need a separate flag to decide whether to clean up or not. Currently, if you do a manual run twice, the images will only be built once right? What if I want to rebuild the second time?